src/hard_spinner.cpp

```diff
@@ -0,0 +1,26 @@
+/**
+ * @file hard_spinner.cpp
+ * @author Julian Viereck (jviereck@tuebingen.mpg.de)
+ * license License BSD-3-Clause
+ * @copyright Copyright (c) 2020, New York University and Max Planck
+ * Gesellschaft.
+ * @date 2020-11-26
+ *
+ * @brief This file implements a hard spinner to time a loop.
+ */
+
+#include <real_time_tools/hard_spinner.hpp>
+#include <real_time_tools/timer.hpp>
+
+namespace real_time_tools
+{
+HardSpinner::HardSpinner()
+: Spinner() { }
+
+void HardSpinner::spin()
+{
+    Timer::sleep_until_sec(next_date_sec_);
```

Comment:
problem:
is 3 tics are missed this method does sleep for 3 times which in a control loop means sending 3 commands without sleeping. Which might mean a robot communication break down.